src/main/java/org/apache/commons/text/StrBuilder.java

```diff
@@ -319,15 +319,14 @@ public StrBuilder append(final boolean value) {
             buffer[size++] = 't';
             buffer[size++] = 'r';
             buffer[size++] = 'u';
-            buffer[size++] = 'e';
         } else {
             ensureCapacity(size + 5);
             buffer[size++] = 'f';
             buffer[size++] = 'a';
             buffer[size++] = 'l';
             buffer[size++] = 's';
-            buffer[size++] = 'e';
         }
+        buffer[size++] = 'e';
```

Comment:
I prefer the other version, even though this one indeed reduces the duplication, the other version clearly shows that we have `true` and `false`, without the dev having to think why `tru` and `fals`, just to find the `e` was set out the `if/else` statement.

Reply:
HI @kinow 
sound fair. Changed
TY

Reply:
I agree with @kinow, you have to think about maintainers reading the code, not just applying changes mechanically.

Reply:
Yep. Agree.